{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2019-20432",
      "description": "In the Lustre file system before 2.12.3, the mdt module has an out-of-bounds access and panic due to the lack of validation for specific fields of packets sent by a client. mdt_file_secctx_unpack does not validate the value of name_size derived from req_capsule_get_size.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2019-20432",
      "reportURL": "https://jira.whamcloud.com/browse/LU-12604",
      "timeStamp": "2020-01-28T16:10:00Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2019-20432",
      "url": "https://lustre.org/",
      "added_at": "1970-01-01T00:00:00Z"
    },
    {
      "id": "CVE-2019-20432",
      "url": "https://wiki.lustre.org/Lustre_2.12.3_Changelog",
      "added_at": "1970-01-01T00:00:00Z"
    }
  ],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2019-20432",
      "url": "https://jira.whamcloud.com/browse/LU-12604",
      "added_at": "2020-01-28T16:10:00Z",
      "score": 0,
      "template_for": ""
    },
    {
      "cve": "CVE-2019-20432",
      "url": "https://lustre.org/",
      "added_at": "1970-01-01T00:00:00Z",
      "score": 0,
      "template_for": ""
    },
    {
      "cve": "CVE-2019-20432",
      "url": "https://wiki.lustre.org/Lustre_2.12.3_Changelog",
      "added_at": "1970-01-01T00:00:00Z",
      "score": 0,
      "template_for": ""
    }
  ]
}
